modules/social_features/social_comment_upload/social_comment_upload.module

```diff
@@ -71,13 +72,17 @@ function social_comment_upload_preprocess_comment(&$variables) {
 
         // Add the amount of attachments to the comment.
         if (!empty($count_arr)) {
-          $text = \Drupal::translation()->formatPlural(count($count_arr),
-            '1 attachment.',
```

Comment:
@rollki If we remove some strings we need to add them to `translations.php` to keep translations for older versions.